### ISSUE TITLE
Disable tabs inside <pre><code> blocks

### ISSUE THREAD

**Issue by zutrinken** [vanetix/grunt-includes#34]
Hi,

I have an issue with tabs inside `<pre><code></code></pre>` tags.

My included file looks like:

	<pre><code>&lt;div class=&quot;yay&quot;&gt;
		&lt;span class=&quot;nay&quot;&gt;
			&lt;!— —&gt;
		&lt;/span&gt;
	&lt;/div&gt;
	</code></pre>

And the generated file looks like:

				<pre><code>&lt;div class=&quot;yay&quot;&gt;
					&lt;span class=&quot;nay&quot;&gt;
						&lt;!— —&gt;
					&lt;/span&gt;
				&lt;/div&gt;
				</code></pre>

Removing/ignoring the superfluous tabs inside `<pre><code></code></pre>` might be very helpful.

_Or is there already an option for this?_

**Comment by vanetix** [comment]
Could you try changing the tabs for spaces? This might be an issue with the way I add indents to lines.

**Comment by zutrinken** [comment]
I did, but instead of extra added tabs (caused by the document structure) I got extra spaces.

But I found my problem. My dest structure looked like:

	<html>
		<body>
			<main>
				„demo.html“
			</main>
		</body>
	</html>

And all I had to do was to change it to:

	<html>
		<body>
			<main>
	„demo.html“
			</main>
		</body>
	</html>

Quite obvious I guess (now). Of course the generated file isn’t formatted as ordered as before but it works properly.

Even if there isn’t a necessary need to fix that for `<pre><code>` blocks, maybe this issue is still worth to mention.
